src/mame/drivers/vii.cpp

```diff
@@ -609,90 +635,123 @@ void vii_state::switch_bank(uint32_t bank)
 	}
 }
 
-void vii_state::do_gpio(uint32_t offset)
+uint16_t spg2xx_cart_state::do_spg243_vii_io(uint16_t what, int index)
 {
-	uint32_t index  = (offset - 1) / 5;
-	uint16_t buffer = m_io_regs[5*index + 2];
-	uint16_t dir    = m_io_regs[5*index + 3];
-	uint16_t attr   = m_io_regs[5*index + 4];
-	uint16_t special= m_io_regs[5*index + 5];
+	if(index == 1)
```

Comment:
Could you please make the spacing consistent with the other if statements (i.e. space between `if` and opening parenthesis).